build.sh

```diff
@@ -53,10 +53,19 @@ sed -i -e "s/BUILD_DATE/${DATE}/g" maestroutils/status.go
 # highlight errors: https://serverfault.com/questions/59262/bash-print-stderr-in-red-color
 # color()(set -o pipefail;"$@" 2>&1>&3|sed $'s,.*,\e[31m&\e[m,'>&2)3>&1
 
+#Developers may have a GOBIN defined. If not define it to current dir.
+if [ -z "$GOBIN" ]
```

Comment:
A monir nit since the pattern already exists in this file.

```suggestion
if [ -z "$GOBIN" ]; then
```